### PR TITLE
Update pstn-usage-report.md

### PR DESCRIPTION
**Pull request by ruizoscar**
Clarifying the user that is specified in the User ID column for these call types

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/skype-for-business-online-reporting/pstn-usage-report.md [hunk 1]
```diff
@@ -66,8 +66,8 @@ This is what the report looks like.
      *    **user_out_forwarding** (the user forwarded the call to a PSTN number)
 
      **Audio Conferencing Call Types**
-     *    **conf_in** (an inbound call to the Audio Conferencing bridge) 
-     *    **conf_out** (an outbound call from the Audio Conferencing bridge usually to add a PSTN number to the conference)
+     *    **conf_in** (an inbound call to the Audio Conferencing bridge). For records of this call type, the user specified in the **User ID** column corresponds to the organizer of the meeting.
+     *    **conf_out** (an outbound call from the Audio Conferencing bridge usually to add a PSTN number to the conference). For records of this call type, the user specified in the **User ID** column corresponds to the organizer of the meeting.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/skype-for-business-online-reporting/pstn-usage-report.md]:
```suggestion
     *    **conf_out** (an outbound call from the Audio Conferencing bridge, usually to add a PSTN number to the conference). For records of this call type, the user specified in the **User ID** column corresponds to the organizer of the meeting.
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**